modules/ROOT/pages/munit-maven-plugin.adoc

```diff
@@ -505,7 +505,7 @@ You can also configure this from the `pom.xml` configuration:
 </plugins>
 ----
 
-You can specify more than one tag by separating them using a comma.
+You can specify more than one tag by separating them with a comma.
```

Comment:
In the MUnit docs, there are instances of "them," "it," etc. that could be clarified, for example:

To specify multiple tags, separate each tag with a comma.

Reply:
goal is always to help people understand what "them," "it," etc. refer to even if it might seem obvious to us.

Reply:
I never had the chance to go over the MUnit docs to correct its content. 
Nice to have, but I don't think I can do it for June 4. 
Thanks for mentioning it!